app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, origin, dupe = command.split()
```

Comment:
Code Style: The variable names `origin` and `dupe` could be more descriptive. Consider renaming them to `source_file` and `destination_file` respectively.